docs/arch/adr-xx.md

```diff
@@ -0,0 +1,65 @@
+  - Some of the drawing tools use _LHC_
+- Some tools' annotation use `angle` some use `rotation`
+- It's unclear when the x, y annotation refers to the center point of the shape
+- It's unclear when the x, y annotation is being used as the point of rotation
+
+Some of the mark annotation models have a few other issues as well:
+
+- Some shapes have default values which an create bias. For example, the ellipse has a default axis ratio of 0.5 and many volunteers have left the default creating a bias ([comment](https://github.com/zooniverse/front-end-monorepo/issues/500#issuecomment-516788821))
+- The freehand drawing tool has peformance impact on the browser as the drawing is being created and with the job to create classification exports as well. This is because the current annotation consists of every single x, y point created
+
+## Decision
+
+The shape's mark annotation models should change for consistency and improved post-classification analysis in the following ways:
+
+- The annotation should use the mathematical standard of _RHC_ with a domain of `(-180, 180]` or `[0, 360)` for consistent angle calculation
+- The annotation model should use `angle` for naming
+- The annotation model should consistency use `x_center` and `y_center` for shapes where that is applicable
+  - The exceptions are point, line, fan. The fan tool is not symmetric. 
+  - Shape clustering in aggregation is always done with the center
+- All rotations should be defined about `x_center` and `y_center` point. If the rotation cannot be defined around the center point, then the point used that should be clearly recorded in the annotation as `x_rotation` and `y_rotation`
+- Default values should be removed wherever possible. This may result in some tools drawing UX changing. 
+  - The tools that have defaults are ellipse, rotate rectangle, fan. 
+  - A proposed example for the ellipse may involve requiring the volunters to click twice to set the the two radii. ([comment](https://github.com/zooniverse/front-end-monorepo/issues/500#issuecomment-516788821))
+- The freehand drawing tools mark annotation will be a string of the SVG's path and it will be the responsibility of post-classification analysis to convert this to usable x,y points. We will include a sample script in the `DataDigging` repo for project owners to reference on how to do this. Aggregation in Caesar will have to be updated to do the conversion first.
+- An added `toolType` string name definition will be included. This would be `point`, `ellipse`, etc. This enables the aggregation for Caesar code to auto-configure which extractor to use without checking data types ([comment](https://github.com/zooniverse/front-end-monorepo/issues/823#issuecomment-493896524)).
+  - `tool` will change to `toolIndex` to clarify it is referring to the index of the input from the task area.
```

Comment:
From memory, `tool` is only ever used to look up the type of tool in `task.tools` so `toolType` would remove it completely.

Reply:
In #1372 `mark.tool` is a reference to the tool model that created this mark, so 👍 from me to this change.

Reply:
It is possible to have multiple instances of the same `toolType` within a drawing task, so `toolIndex` will still be needed.

Example: there are two point tools, one red, the other blue.  These will both have the same `toolType` (point), but should *not* be aggregated together.  Penguin Watch and Seabird Watch are both set up like this.

Reply:
Good point. In the code we have to identify a tool by type and colour, because tools don't have identifiers.
https://github.com/zooniverse/front-end-monorepo/blob/6e686bf20dbf50f05490b98ace7bb0133d788e4e/packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.js#L79-L80

Reply:
One question this raised for me: a project owner can change the order of drawing tools, or even remove some, in the project builder. Is this a rare enough event that we can rely on the array order to identify each tool?

Reply:
I think they could use workflow versions in later analysis to confirm which tool maps to which array index?

Reply:
Yeah, I think workflow version plus array index should work as a tool identifier. Array indexes on their own as identifiers still makes me nervous.